v7/wordpress_compiler/wordpress/default_filters.py

```diff
@@ -0,0 +1,362 @@
+# A WordPress compiler plugin for Nikola
+#
+# Copyright (C) 2014-2015 by the contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
```

Comment:
I took the original copyright notice found in WordPress source files, and I took their `license.txt` (and added a part for this plugin).